src/components/ServiceList.jsx

```diff
@@ -1,22 +1,57 @@
 import { Alert } from "@baltimorecounty/dotgov-components";
 import FilterList from "../components/FilterList";
-import { MostPopularServiceIconStyles } from "../styles";
-import React from "react";
-import ServiceIconLink from "../components/ServiceIconLink";
+import React, { useState } from "react";
+import ServiceIconLink from "./ServiceIconLink";
+import PopularityFilterCollapse from "./PopularityFilterCollapse";
 import useServices from "../hooks/useServices";
+import { MostPopularServiceIconStyles } from "../styles";
+import { TextInput } from "@baltimorecounty/dotgov-components";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isMostPopular, setMostPopular] = useState(false);
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [filterText, setFilterText] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(false);
 
-  let legendText = {
-    fontStyle: "italic",
-    marginLeft: "5px",
-    fontWeight: "900"
+  const filterItems = (services, searchText) => {
```

Comment:
think this can be moved outside the component.

Reply:
moved